src/StateLabel.tsx

```diff
@@ -37,7 +77,7 @@ const StateLabelBase = styled.span<StyledStateLabelBaseProps>`
   align-items: center;
   font-weight: 600;
```

Comment:
While we're editing this file might as well use theme values here :) 
```suggestion
  font-weight: ${get('fontWeights.bold')};
```